src/main/kotlin/nl/avisi/kotlinwebtest/rest/Validation.kt

```diff
@@ -25,18 +28,26 @@ enum class CompareMode {
             }
 }
 
-class JsonValidator(val mode: CompareMode, var value: Expression? = null) : Validator<RestRequest, RestStepResponse> {
+class JsonValidator(val mode: CompareMode, var value: Expression? = null, vararg val pathAndRegex : Pair<String, String>) : Validator<RestRequest, RestStepResponse> {
```

Comment:
Would be useful to be able to test against a dynamic value (using an expression), as alternative to a regex.